docs/buildah-add.md

```diff
@@ -47,5 +47,52 @@ buildah add containerID 'https://github.com/containers/buildah/blob/master/READM
 
 buildah add containerID 'passwd' 'certs.d' /etc
 
+## FILES
+
+### `.dockerignore`
+
+If the file .dockerignore exists in the context directory, `buildah copy` reads 
+its contents. Buildah uses the content to exclude files and directories from 
+the context directory, when copying content into the image.
+
+Users can specify a series of Unix shell globas in a .dockerignore file to
```

Comment:
```suggestion
Users can specify a series of Unix shell globals in a .dockerignore file to
```

Reply:
be nice if we could get .containerignore working to or instead of .dockerignore.

Reply:
Yes I thought we had done this.  Need a different PR.